frontend/src/app/config.js

```diff
@@ -1,4 +1,6 @@
 export default {
   experimentsURL: '/api/experiments.json',
-  usageCountsURL: 'https://analysis-output.telemetry.mozilla.org/testpilot/data/installation-counts/latest.json'
+  usageCountsURL: 'https://analysis-output.telemetry.mozilla.org/testpilot/data/installation-counts/latest.json',
+  productionURL: 'https://testpilot.firefox.com',
+  stagingURL: 'https://testpilot.stage.mozaws.net'
```

Comment:
We should stash this somewhere that gets copied to both the addon and site in the build process. Silly to define it twice.